nixos/tests/k3s/default.nix

```
@@ -7,28 +7,56 @@ let
   allK3s = lib.filterAttrs (
     n: _: lib.strings.hasPrefix "k3s_" n && (builtins.tryEval pkgs.${n}).success
   ) pkgs;
+
+  tests =
+    (selfTests: {
+      all = lib.mapAttrs (
+        _: k3s:
+        let
+          k3sKeyVersion = "k3s_" + (lib.versions.major k3s.version) + "_" + (lib.versions.minor k3s.version);
+        in
+        pkgs.buildEnv {
```

```suggestion
        pkgs.buildEnv {
```
I don't know if there is a better option for this than `buildEnv` that would show each test build in separate.